### PR TITLE
[SC] print.c: Minor code improvements

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Proposed changes

- Plus: Update license header.
- Re-use PrintServiceStatus(), as SERVICE_STATUS_PROCESS is in fact an extension of SERVICE_STATUS.
- Use the "SERVICE_*" defines, not magic numbers.

### CODE REVIEW

File: base/applications/sc/print.c [hunk 1]
```diff
@@ -110,50 +36,63 @@ PrintServiceStatus(
...
         case SERVICE_WIN32_SHARE_PROCESS + SERVICE_INTERACTIVE_PROCESS:
-            _tprintf(_T("WIN32_SHARE_PROCESS (interactive)\n"));
+            _tprintf(_T("WIN32_SHARE_PROCESS (interactive)"));
             break;
-
-        default : _tprintf(_T("\n")); break;
+        // default: // Nothing.
     }
 
+    _tprintf(_T("\n"));
```
Comment (sanchaez) [inline base/applications/sc/print.c]: It is an unnesessary change as it adds an additional `_tprintf()` call and looks more ugly.

File: base/applications/sc/print.c [hunk 2]
```diff
@@ -110,50 +36,63 @@ PrintServiceStatus(
...
+        case SERVICE_PAUSE_PENDING:
+            _tprintf(_T("PAUSE_PENDING"));
+            break;
+        case SERVICE_PAUSED:
+            _tprintf(_T("PAUSED"));
+            break;
+        // default: // Nothing.
     }
 
+    _tprintf(_T("\n"));
```
Comment (sanchaez) [inline base/applications/sc/print.c]: ditto

File: base/applications/sc/print.c [hunk 3]
```diff
@@ -1,10 +1,9 @@
 /*
  * PROJECT:     ReactOS Services
- * LICENSE:     GPL - See COPYING in the top level directory
- * FILE:        base/applications/sc/print.c
```
Comment (sanchaez) [inline base/applications/sc/print.c]: This line should be kept.

Comment (SergeGautherie) [reply base/applications/sc/print.c]: Reverted.

File: base/applications/sc/print.c (hunk 2 above)
Comment (SergeGautherie) [reply base/applications/sc/print.c]: Reverted.

File: base/applications/sc/print.c (hunk 1 above)
Comment (SergeGautherie) [reply base/applications/sc/print.c]: Reverted.

File: base/applications/sc/print.c [hunk 4]
```diff
@@ -1,10 +1,10 @@
 /*
  * PROJECT:     ReactOS Services
- * LICENSE:     GPL - See COPYING in the top level directory
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
  * FILE:        base/applications/sc/print.c
```
Comment (HBelusca) [inline base/applications/sc/print.c]: Technically speaking the new code header style does not require anymore the "FILE:" line (see ["File Structure"](https://www.reactos.org/wiki/Coding_Style#File_Structure)), so you can either keep it or remove it (I don't really mind).

**Comment by HBelusca** [comment]
@SergeGautherie @sanchaez: let us know when everything is ready :)

File: base/applications/sc/print.c [hunk 5]
```diff
@@ -1,10 +1,10 @@
 /*
  * PROJECT:     ReactOS Services
- * LICENSE:     GPL - See COPYING in the top level directory
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
```
Comment (learn-more) [inline base/applications/sc/print.c]: If you change the license, you are required to ask all authors of the file for their agreement.

Comment (gedmurphy) [reply base/applications/sc/print.c]: fine by me :)

**Pull request merged**